### PR TITLE
Added options to switch text font size and position

### LINKED ISSUE #1107: Option to increase the attributes text size
Sometimes I find difficult to check the text attributes of the box, It would be good to give an option to customize the font size, for now, I have added  some CSS in the HTML 👇 
```
.shapeText {
    font-size: 0.3em;
    fill: white;
    stroke:black;
    stroke-width: 0.2;
    cursor: default;
}
```
image

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context
image
 
Default behaviour:
image

Updated behaviour:
image

Polylabel algorithm is used to find best place for text (https://blog.mapbox.com/a-new-algorithm-for-finding-a-visual-center-of-a-polygon-7c77e6492fbc)

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

Resolved #1107

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
Hi, @nstolyarov

Could you please look at this PR and say us if this is a thing could help you, or maybe suggest other ideas?

File: cvat-canvas/src/typescript/consts.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ const ARROW_PATH = 'M13.162 6.284L.682.524a.483.483 0 0 0-.574.134.477.477 0 ' +
 const BASE_PATTERN_SIZE = 5;
 const SNAP_TO_ANGLE_RESIZE_DEFAULT = 0.1;
 const SNAP_TO_ANGLE_RESIZE_SHIFT = 15;
+const DEFAULT_SHAPE_TEXT_SIZE = 14;
```
Comment (ActiveChooN) [inline cvat-canvas/src/typescript/consts.ts:22]: Is this const used anywhere?

Comment (bsekachev) [reply cvat-canvas/src/typescript/consts.ts]: It was not, but now it is used. Thanks.

**Review by nmanovic — APPROVED** [review]
LGTM, I believe the polylabel is a great solution for the problem.

**Review by ActiveChooN — APPROVED** [review]

**Comment by bsekachev** [comment]
@aschernov @TOsmanov 

Guys, could you please update documentation according to these new features?

**Comment by bsekachev** [comment]
@dvkruchinin Could you please prepare a test? At least font-size should be tested. I am not sure about good approach to test position, but if you have any ideas how it could be implemented, it is great

**Comment by TOsmanov** [comment]
> @aschernov @TOsmanov
> 
> Guys, could you please update documentation according to these new features?

@bsekachev Yes, sure

**Comment by dvkruchinin** [comment]
> @dvkruchinin Could you please prepare a test? At least font-size should be tested. I am not sure about good approach to test position, but if you have any ideas how it could be implemented, it is great

@bsekachev Sure. A test will be prepared.

**Pull request merged**